cmake/Deployment.cmake

```diff
@@ -10,5 +10,8 @@
 
 set(LY_ASSET_DEPLOY_MODE "LOOSE" CACHE STRING "Set the Asset deployment when deploying to the target platform (LOOSE, PAK, VFS)")
 set(LY_ASSET_OVERRIDE_PAK_FOLDER_ROOT "" CACHE STRING "Optional root path to where Pak file folders are stored. By default, blank will use a predefined 'paks' root.")
-
-
+set(LY_ARCHIVE_FILE_SEARCH_MODE "" CACHE STRING "Set the default file search mode to locate non-Pak files within the Archive System\n\
+    Valid values are:\n\
+    0 = Search FileSystem first, before searching within mounted Paks\n\
+    1 = Search mounted Paks first, before searching FileSystem\n\
+    2 = Search only mounted Paks\n")
```

Comment:
here we should state that 2 is the default for release and 0 is the default for debug/profile

Reply:
Will update.